ocaml/lib/snarky_js_bindings_lib.ml

```diff
@@ -1626,13 +1626,12 @@ module Circuit = struct
       Js.wrap_callback (fun (f : (unit -> unit) Js.callback) : unit ->
           Impl.as_prover (fun () -> Js.Unsafe.fun_call f [||]) ) ;
     circuit##.generateKeypair :=
-      Js.wrap_meth_callback
-        (fun (this : _ Circuit_main.t) : keypair_class Js.t ->
-          generate_keypair this ) ;
+      Js.wrap_callback (fun (circuit : _ Circuit_main.t) : keypair_class Js.t ->
+          generate_keypair circuit ) ;
     circuit##.prove :=
-      Js.wrap_meth_callback
-        (fun (this : _ Circuit_main.t) w p (kp : keypair_class Js.t) ->
-          prove this w p kp##.value ) ;
+    Js.wrap_callback
+    (fun (circuit : _ Circuit_main.t) w p (kp : keypair_class Js.t) ->
+      prove circuit w p kp##.value ) ;
```

Comment:
small changes to the OCaml -> JS `Circuit` API because we now wrap this API on the JS side (see https://github.com/o1-labs/snarkyjs/pull/872)